src/cli/mod.rs

```diff
@@ -160,6 +192,36 @@ impl CkbArgs {
     pub fn client(&self) -> CkbRpcClient {
         CkbRpcClient::new(self.ckb_endpoint.as_str())
     }
+
+    pub fn fee_rate(&self) -> Result<u64> {
+        let value = if let Some(fixed) = self.fee_rate.value() {
+            log::info!("CKB fee rate: {} (fixed)", FeeRate(fixed));
+            fixed
+        } else {
+            let dynamic = self.client().dynamic_fee_rate()?;
+            log::info!("CKB fee rate: {} (dynamic)", FeeRate(dynamic));
+            if dynamic > 1_0000_0000 {
+                let msg = format!(
+                    "dynamic CKB fee rate {} is too large, it seems unreasonable;\
+                    if you can confirm it, please set it as fixed fee rate",
+                    FeeRate(dynamic)
+                );
+                return Err(Error::other(msg));
```

Comment:
If the dynamic fee rate exceeds the threshold 1_0000_0000, the service crashes.

Reply:
What making MAX_DYNAMIC_FEE_RATE configurable and defaulting to 1_0000_0000?